WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1866,10 +1841,10 @@ protected Boolean doInBackground(Void... params) {
                     // or a shortcode replacement (for instance for images and galleries)
 
                     // Update the post object directly, without re-fetching the fields from the EditorFragment
-                    updatePostContentNewEditor(postModel, false, postModel.getTitle(), postModel.getContent());
+                    updatePostContentNewEditor(postModel);
```

Comment:
yep, the one that used to be called from here is simplified `updatePostContentNewEditor(postModel, false, postModel.getTitle(), postModel.getContent());` since it was always comparing the actual post title and content with itself (and not updating it). We've discussed splitting the method in 2 on Slack. The original one is now moved to the ViewModel.

Reply:
As discussed on Slack I'd consider renaming one of the methods.

Reply:
Changed in 811b84b067742e57d5a8e2c53056ec1c0d62f913